http/header.go

```diff
@@ -31,6 +32,11 @@ func (header *Header) IsConnectionClose() bool {
 	return header.isConnectionClose
 }
 
+//IsProxyConnectionClose is Proxy-Connection header set to `close`
+func (header *Header) IsProxyConnectionClose() bool {
+	return header.isProxyConnectionClose
```

Comment:
considering the `Proxy-Connection` and `Connection` are [hop by hop headers](http://freesoft.org/CIE/RFC/2068/143.htm), the best solution for this issue is test `Proxy-Connection: close`  in `IsConnectionClose` func.

Reply:
we do NOT care if the connection is close or proxy-connection is close, we only care if we should keep the connection